include/mbedtls/ssl.h

```diff
@@ -976,6 +964,58 @@ struct mbedtls_ssl_session
 #endif
 };
 
+/*
+ * Identifiers for PRFs used in various versions of TLS.
+ */
+typedef enum
+{
+   MBEDTLS_SSL_TLS_PRF_NONE,
+   MBEDTLS_SSL_TLS_PRF_TLS1,
```

Comment:
Is MBEDTLS_SSL_TLS_PRF_TLS1 still needed after TLS 1.0 and 1.1 have been removed?

Reply:
Yes :+1:

Reply:
@daverodgman I think we need to defer this because it breaks interoperability tests with OpenSSL which currently still rely on this constant.